pytorch/inference/docker/1.5.1/py3/Dockerfile.neuron

```diff
@@ -104,11 +103,11 @@ RUN pip install --no-cache-dir -U \
     grpcio==1.24.1 \
     protobuf==3.10.0 \
     gast==0.2.2 \
-    pillow==7.2.0 \
+    pillow==8.2.0 \
     "awscli<2"
 
 RUN pip install neuron-cc[tensorflow] --extra-index-url https://pip.repos.neuron.amazonaws.com \
- && pip install "torch-neuron>=1.5.1,<1.5.2" --extra-index-url https://pip.repos.neuron.amazonaws.com \
+ && pip install "torch-neuron>=1.5.1.1.2,<1.5.1.1.3" --extra-index-url https://pip.repos.neuron.amazonaws.com \
```

Comment:
why is this added?

Reply:
There are multiple builds for `torch-neuron`, and all builds which are `1.5.1.1.2.*` are targeted at ubuntu16.04, while all versions `1.5.1.1.3.*` are targeted at ubuntu18.04, which is why I've constrained the version.